hubblestack/utils/signing.py

```diff
@@ -508,16 +529,56 @@ def manifest(targets, mfname=None):
     if mfname is None:
         mfname = Options.manifest_file_name
 
-    with open(mfname, 'w') as mfh:
+    with open(mfname, "w") as mfh:
+
         def append_hash(fname):
             fname = normalize_path(fname)
             digest = hash_target(fname)
-            mfh.write('{} {}\n'.format(digest, fname))
-            log.debug('wrote %s %s to %s', digest, fname, mfname)
+            mfh.write("{} {}\n".format(digest, fname))
+            log.debug("wrote %s %s to %s", digest, fname, mfname)
+
         descend_targets(targets, append_hash)
 
 
-def sign_target(fname, ofname, private_key=None, **kwargs): # pylint: disable=unused-argument
+def encode_pem(data: bytes, marker: str) -> str:
+    """
+    Encode a bytestring as base64, using the PEM format.
+    :param data: bytestring to be encoded
+    :param marker: PEM format specific header
+    :returns: PEM encoded string
+    """
+    output = "-----BEGIN {}-----\n".format(marker)
+    chunks = [b2a_base64(data[i : i + 48]).decode("latin-1") for i in range(0, len(data), 48)]
+    output += "".join(chunks)
+    output += "-----END {}-----".format(marker)
+    return output
+
+
+def decode_pem(data: bytes) -> str:
```

Comment:
May be able to use this instead: https://cryptography.io/en/latest/x509/reference/#loading-certificates

Reply:
I'm not sure. `PEM.encode` and `PEM.decode` were being used to get a `base64` representation of the key/cert. The functions from `cryptography.x509` are highly specialised in what they do.